src/main/java/com/mathworks/ci/MatlabCommandStepExecution.java

```diff
@@ -72,11 +72,10 @@ private synchronized int execMatlabCommand(FilePath workspace, Launcher launcher
 
         try {
             matlabLauncher = getProcessToRunMatlabCommand(workspace, launcher, listener, envVars,
-                    uniqueCommandFile, uniqueTmpFldrName);
-            launcher.launch().pwd(uniqeTmpFolderPath).envs(envVars);
+                    "cd('"+ uniqeTmpFolderPath.getRemote().replaceAll("'", "''") +"');"+ uniqueCommandFile, uniqueTmpFldrName);
```

Comment:
Typo in `unique`. Maybe add a space between the cd(...); and the call to the script?

Reply:
Done.